apps/evm/test/evm/operation/system_test.exs

```diff
@@ -21,4 +21,32 @@ defmodule EVM.Operation.SystemTest do
       assert Map.get(accounts, selfdestruct_address) == nil
     end
   end
+
+  describe "revert" do
```

Comment:
Looking at these two tests (the one for `revert` on line 26 and the one for `return` on line 40), I can see that they should be have differently, but their assertions are the same. Is there a way we could write these tests so that we can see that difference clearly in the assertions? I don't know enough of the VM code to suggest a possible way to do this, but I'm curious if it can be done.

Reply:
yup, so as I understand the difference is that `:return` doesn't not **revert** the state changes

Reply:
I will find machine code  that changes exection env so assertions will be different

Reply:
@germsvel @vyorkin Now I check state after SSTORE operation. It should change state in case of RETURN op code